apps/marketplace/pages/BriefResponsesPage.js

```diff
@@ -30,8 +30,11 @@ class BriefResponsesPage extends Component {
           this.setState({
             loading: false
           })
+          if (response.data.brief.status !== 'live') {
+            this.props.setError('This opportunity has closed - you can no longer make edits.')
```

Comment:
What about withdrawn opportunities?  Does it make sense to rewrite this to something like the following?

`Edits can't be made on a <status> opportunity`